packages/components/pages/f-loyalty/src/store/loyalty.module.js

```diff
@@ -10,10 +10,31 @@ import {
 /**
  * @function resolveGlobalUserID
  * @param  {string} authToken - auth token.
+ * @param logger
  */
-const resolveGlobalUserID = authToken => {
-    const tokenData = jwtDecode(authToken);
-    return tokenData?.global_user_id;
+const resolveGlobalUserID = (authToken, logger) => {
+    // eslint-disable-next-line no-useless-catch
+    try {
+        const tokenData = jwtDecode(authToken);
+        return tokenData?.global_user_id;
+    } catch (error) {
+        logger.error(
+            'Error Decoding JWT token',
+            error,
+            'loyalty',
+            {
+                ...{
+                    authToken
+                },
```

Comment:
Adding `authToken` directly here should achieve the same thing without the spread I believe? 

```suggestion
                authToken,
```

Reply:
thanks fixed